README.md

```diff
@@ -1,29 +1,234 @@
 # profiles
 Gitops native package management
 
-## Installation on a local cluster using [Kind](https://kind.sigs.k8s.io/)
+<!--
+To update the TOC, install https://github.com/kubernetes-sigs/mdtoc
+and run: mdtoc -inplace README.md
+-->
+
+ <!-- toc -->
+- [Getting started](#getting-started)
+  - [Local environment using <a href="https://kind.sigs.k8s.io/">Kind</a>](#local-environment-using-kind)
+  - [Local development](#local-development)
+  - [Installing Profiles](#installing-profiles)
```

Comment:
feels like some sort of intro & tldr is missing, going straight into local-env setup is jumping the gun I think

Reply:
yeh i will add more in another PR